eq-author/src/components/ContentPickerv2/DestinationPicker/Menu.test.js

```diff
@@ -126,131 +134,173 @@ function defaultSetup() {
 }
 
 describe("Destination Picker Menu", () => {
-  it("should default to current section tab", () => {
-    const { getByText } = defaultSetup();
-    expect(getByText("Question one")).toBeVisible();
-  });
+  describe("Hub is disabled", () => {
```

Comment:
```suggestion
  let questionnaire;
  
  describe("Hub is disabled", () => {
```

Needs the above change as well; this moves the statement closer to the thing that initialises it for readability.